driver/src/orderbook/event_based_orderbook/event.rs

```diff
@@ -0,0 +1,61 @@
+use super::*;
+use ethcontract::U256;
+
+#[derive(Clone, Copy, Debug)]
+pub enum Event {
```

Comment:
This will hopefully be generated by `ethcontract`. Although, the format would be slightly different:
```
enum EventData {
    Deposit(Deposit),
    Withdraw(Withdraw),
    // ...
}
```

This is because we reuse the data in the `all_events` and `event` streams.